### ISSUE TITLE
Support "skipped_deps" as a raze.crate param.

### ISSUE THREAD

**Issue by acmcarther** [acmcarther/cargo-raze#20]
I have a situation with the sdl2 crate where I really need to replace `sdl2-sys` with my own bindgen'd crate. It is possible to add a new crate to serve the purpose, but it isn't possible to remove the old one.

**Comment by acmcarther** [comment]
The form it should take should be similar to additional_deps:
```
skipped_deps = [
  "//cargo/vendor/sdl2-sys-0.31.0:sdl2_sys"
]
```

**Comment by acmcarther** [comment]
This was implemented in #21 directly.

In the example, I dropped the (critical) regex-syntax dep from the regex crate. The effect is visible in the generated BUILD file:

Cargo.toml
https://github.com/acmcarther/cargo-raze-examples/blob/a0bb06c5833bb35e2391b6306e9d83d4b7bc2e80/bazel/complicated_cargo_library/Cargo.toml#L17

regex/BUILD
https://github.com/acmcarther/cargo-raze-examples/blob/a0bb06c5833bb35e2391b6306e9d83d4b7bc2e80/bazel/complicated_cargo_library/vendor/regex-0.2.5/BUILD#L30

EDIT: Or not.... Looks like regex was surreptitiously upgraded, and my override ceased to apply. Need to make that mistake harder to make. Fixed commit:

https://github.com/acmcarther/cargo-raze-examples/commit/1d77918e7a86e6a57508feac30837b5741e27c81
